HW02_B_Mimic_starter.Rmd

```diff
@@ -41,7 +41,17 @@ data("diamonds")
 
 Using the diamonds dataset, make this graph:
 ```{r graph1 code, echo=FALSE}
-
+ggplot(diamonds,aes(cut, fill=clarity)) + 
+#Make bars dodged side-to-side
+geom_bar(position="dodge") + 
+#Create labels for x-axis, y-axis, title, and subtitle
+labs(x="Diamond Cut", y="Number of Diamonds", title="My Diamond Collection", subtitle="Boxplot representing the number of diamonds in my diamond collection by \n type of cut quality and clarity of diamond") +
+#Create a rectangle annotate

```

Comment:
I used the same approach, but I wonder if there is a way to automatically change line.